examples/webgl_loader_fbx.html

```diff
@@ -30,13 +30,29 @@
 
 			import { OrbitControls } from 'three/addons/controls/OrbitControls.js';
 			import { FBXLoader } from 'three/addons/loaders/FBXLoader.js';
+			import { GUI } from 'three/addons/libs/lil-gui.module.min.js';
 
-			let camera, scene, renderer, stats;
-
+			let camera, scene, renderer, stats, object, loader, guiMorphsFolder;
 			const clock = new THREE.Clock();
 
 			let mixer;
 
+			const params = {
+				asset: 'Samba Dancing'
+			};
+
+			const assets = [
+				'Samba Dancing',
+				'morph_test'
+			];
+
+			const transforms = {
```

Comment:
Do you think you can update `morph_test.fbx` such that no additional transforms are required in the example? Ideally the asset does not need be scaled and translated after the import. That would simplify the example a bit.

Reply:
Yes I can, however, the existing rabbit is already huge. Should we let it like that ?

Reply:
I could, re-export the rabbit too to make it have a reasonable size

Reply:
The rabbit is used in other examples I would not like to touch them now. Notice that I have updated your branch and removed the nurbs and rabbit asset from the demo. They are already used in other demos so in `webgl_loader_fbx`  I would only add assets which are not used elsewhere.

Reply:
I see you removed the rabbit and nurbs. 
Either we don't show it or I can re-export it (I'll do it in blender if so).
Meanwhile I'll just re-export the test example.

Reply:
Pushed, changes.

Reply:
The CI / E2E testing fail I think it was random. I tested manually that page (webgl_animation_skinning_ik) and looks fine (and it uses a gltf asset anyways).